### DISCUSSION TITLE
blog/you-dont-need-a-plugin-to-customize-shopware-6/

### DISCUSSION THREAD

**Discussion by giscus** [shyim/shyim.me#51] [Q&A]
# blog/you-dont-need-a-plugin-to-customize-shopware-6/

Every developer who has created a plugin has already created a Symfony bundle. All Shopware plugins are extending from a Shopware bundle class which extends from the Symfony bundle. You might now ask yourself: when plugins are bundles, why did you write this blog post? First, let’s list the differences of a plugin and the two types of bundles:
   Feature Plugin Shopware Bundle Symfony Bundle     modifying Shopware ✔️ ✔️ ✔️   can have migrations ✔️ ✔️ ❌   can be a theme ✔️ ✔️ ❌   can modify admin / storefront with js/css ✔️ ✔️ ❌   can be installed, uninstalled (lifecyle) ✔️ ❌ ❌   can be managed by the shop owner in administration ✔️ ❌ ❌    The benefits of working with a Bundle Having less features is sometimes better.

https://shyim.me/blog/you-dont-need-a-plugin-to-customize-shopware-6/

**Comment by anewarray** [comment]
is this still valid  with shopware 6.5?

**Reply by shyim** [reply]
Yes. You can do it in any Shopware 6 version
